Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -55,6 +51,53 @@ async function run(): Promise<void> {
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
 
+#if WIF
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
+        const feedIdNames = tl.getDelimitedInput("artifactsFeeds", ',');
+
+        if (entraWifServiceConnectionName) {
+
+            if (feedIdNames.length === 0) {
+                tl.warning(tl.loc("Warning_NoEndpointsToAuth"));
+            }
+            
+            tl.debug(tl.loc("Info_AddingFederatedFeedAuth", entraWifServiceConnectionName));
+            let token = await getFederatedWorkloadIdentityCredentials(entraWifServiceConnectionName);
+            
+            if (token) {
+
+                for (let feedName of feedIdNames) {
```

Comment:
Can this be moved to mavenUtils similar to `getInternalFeedsServerElements` and `getExternalServiceEndpointsServerElements`?

Reply:
If we structured this like mavenUtils methods, I think it would actually cause the task a little more work if it had to go through a list of feed names and then go through another list of server elements, rather than do everything at once